README.md

```diff
@@ -1,49 +1,99 @@
-# Software Stack
 
 ![Ubuntu](https://github.com/noaa-emc/hpc-stack/workflows/Build%20Ubuntu/badge.svg)
 ![macOS](https://github.com/noaa-emc/hpc-stack/workflows/Build%20macOS/badge.svg)
 
-This repository provides a unified, shell script based build system for building commonly used software stack.
+# hpc-stack
+
+This repository provides a unified, shell script based build system
+for building software stack needed for the NOAA [Universal Forecast System
+(UFS)](https://github.com/NOAA-GSD/ufs-weather-model) and related
```

Comment:
Please use `https://github.com/ufs-community/ufs-weather-model`  The one being pointed to currently is a fork.

Reply:
Done.